README.md

```diff
@@ -4,6 +4,25 @@
 ---
 Retro chiptune music is starting to make a resurgence with the popularity of many retro indie games like Hades and Celeste. One of the reasons why it's gaining its popularity again is because it resonates with a lot of people; it brings back nostalgia and fond memory associated with it. However, a majority of music produced for these games rarely use actual hardware that made the 8-bit and 16-bit music charming. The purpose of this project is to replicate a sound chip from the NES (Nintendo Entertainment System) using Rust, and bring back those memories to the users.
 
+## Milestones
+---
+### Milestone 1 (worst case scenario):
+>* Basic audio generator that can produce 8-bit sound based on user's keyboard input.
+
+### Milestone 2 (expected):
+>* The program can accept MIDI file as an input, and convert it to chiptune sound. It should be able to handle multiple channels as well.
+Various MIDI files should be used for testing this function, such as relatively simple classical music file with a single channel, and more complex pop song with multiple channels. <br /><br />
```

Comment:
Part of the first sentence should be changed to "such as a relatively simple classical music file with a single channel, and more complex pop songs with multiple channels."